### ISSUE TITLE
Got better results on Faster R-CNN when trained with smaller LR/batch

### ISSUE THREAD

**Issue by ArieSeirack** [open-mmlab/mmdetection#4993]
In original COCO 1x schedule configuration, the learning rate is 0.02 for 8 GPUs and 2 imgs per GPU.
I have one Titan XP GPU now. 
So following the Linear Scaling Rule, I changed the learning rate to 0.0025 and kept 2 imgs per GPU.
Then I got a better result than your faster_rcnn_r50_fpn_1x_coco model. I did not change any other configurations.

The following data format is (mAP | AP50 | AP75 | AP_small | AP_medium | AP_large ). 
My result is 
37.6 | 58.4 | 40.7 | 22.1 | 41.4 | 48.3

And your model result is 
37.4 | 58.1 | 40.4 | 21.2 | 41.0 | 48.1

Is it resonable to get such result?

**Comment by shinya7y** [comment]
The Linear Scaling Rule does not produce exactly the same results.
E.g., Figure 1 of https://www.ecva.net/papers/eccv_2020/papers_ECCV/papers/123660477.pdf

**Comment by ArieSeirack** [comment]
> The Linear Scaling Rule does not produce exactly the same results.
> E.g., Figure 1 of https://www.ecva.net/papers/eccv_2020/papers_ECCV/papers/123660477.pdf

Thanks. 
So following Linear Scaling Rule, small learning rate with small batch size is able to have better performance than large learning rate with large batch size.
